ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/util/SWTUtil.java

```diff
@@ -106,4 +106,8 @@ public static void setDialogSize(Dialog dialog, int width, int height) {
 		height = Math.max(computedSize.y, height);
 		dialog.getShell().setSize(width, height);
 	}
+
+	public static boolean isBidiPlatform() {
```

Comment:
Isn't org.eclipse.jface.util.BidiUtils.getBidiSupport() the API for that?

Reply:
Would be good if that public API could be used here. However the implementation is totally different. jface is a preference setting while swt is some platform thing. I don't know what is actually required here and don't want a functional change with this refactoring

Reply:
Makes sense.